### ISSUE TITLE
Deferred Deep Link NSInvalidArgumentException

### ISSUE THREAD

**Issue by toddtreece** [singular-labs/react-native-sdk#10]
We are seeing crashes on iOS when using deferred deep links due to this error:
```
Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** -[__NSPlaceholderDictionary initWithObjects:forKeys:count:]: attempt to insert nil object from objects[1]'
```

It looks like `getPassthrough` returns null:
https://github.com/singular-labs/react-native-sdk/blob/53f5bde74b9eefb49ff480f1c486c869dd875f75/ios/SingularBridge.m#L198

If I set it to a static string the app no longer crashes:
```
@"passthrough": @"test",
```

**Comment by eyalsingular** [comment]
Hi @toddtreece,

Thank you very much for notifying us!
We have fixed the issue and released a new version 1.1.1.

Cheers,
Eyal

**Comment by toddtreece** [comment]
thanks!
